src/main/scala-2.13-/scala/concurrent/stm/compat/Proxy.scala

```diff
@@ -0,0 +1,48 @@
+package scala.concurrent.stm
```

Comment:
we should add this to https://github.com/scala/scala-collection-compat

Reply:
No, we should not use these proxies because they are deprecated. The goal of scala-collection-compat is not to provide 2.12 syntax on 2.13, but to provide 2.13 syntax on 2.12.

Reply:
https://github.com/scala/bug/issues/11009